README.md

```diff
@@ -1,8 +1,30 @@
 # Northcoders News API
 
+Project Summary: This is a NEWS API with endpoints to access news articles, users and comments on articles.
+It is designed to be able to access app data programmatically in order to provide this data to front-end architecture.
+
+Link to hosted repo: https://my-nc-news-s3ph.onrender.com/
```

Comment:
Might be better to link to /api so that if they click the link they see something